pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["flit"]
+build-backend = "flit.buildapi"
+
+[tool.flit.metadata]
+module = "py-datastore"
+#author       = ""
+#author-email = ""
+home-page    = "https://github.com/ipfs/py-datastore"
+keywords     = "ipfs datastore database development"
+license      = "MIT License"
+description-file = "README.md"
+
+# Unfortunately these currently need to be duplicated from `requirements.txt`
+requires-python = ">=3.7"
+
+# TODO
+requires = [
+	''
+]
```

Comment:
I just checked and the `datastore` package really only depends on the stdlib. So, this list being empty is totally fine.

Also the file `requirements_all.txt` can be removed as it lists dependencies of the submodules, rather than on the main package.